core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -3925,6 +3926,28 @@ public void checkOrderBy(final boolean desc,
         .returnsUnordered("empid=150; name=Sebastian");
   }
 
+  /** Test case for <a href="https://issues.apache.org/jira/browse/CALCITE-6880">
```

Comment:
this comment seems wrong
why does this case fail? 
does it invoke the new rule?

Reply:
Yes, the new rule can cause this problem. Should I keep this case and modify the comments?